core/tauri/src/window.rs

```diff
@@ -1301,6 +1314,11 @@ impl<R: Runtime> Window<R> {
       .map_err(Into::into)
   }
 
+  /// Sets window effects, pass `None` to clear any effects applied if possible.
+  pub fn set_window_effects(&self, effects: Option<WindowEffectsConfig>) -> crate::Result<()> {
```

Comment:
I think we should provide a Builder for the window effects, it would be easier to use (with a `From<WindowsEffectsConfig>` impl to simplify the window plugin and if someone wants to use the raw config object, which is unstable btw, if we add new fields it would break users apps unless they use `..Default::default()`)

Reply:
added in [43a2e71](https://github.com/tauri-apps/tauri/pull/6442/commits/43a2e717ef53199629733ba031abfd242575936d)